pages/curriculum.tsx

```diff
@@ -111,5 +150,20 @@ export const Curriculum: React.FC<{}> = () => {
     </Layout>
   )
 }
+const FIVE_MINUTES = 5 * 60
+
+export const getStaticProps: GetStaticProps = async () => {
+  const apolloClient = initializeApollo()
+  const query = await apolloClient.query<GetAppQuery>({
+    query: GetAppDocument
+  })
 
+  return {
+    props: {
+      lessons: query.data.lessons,
+      alerts: query.data.alerts
```

Comment:
I think you can leave the alerts out of the SSG. They're not that common anyway. Might also cause some issues with alerts rendering on the page load making they show up and being dismissed after the page scripts run.

Reply:
Yes, I was thinking about it too. With this setup alerts will be updated every five minutes. Considering how often we use them it's not really a big deal. If it causes any problems with the rendering I'll include it in apollo query.